pkgs/by-name/un/unl0kr/package.nix

```diff
@@ -1,30 +1,33 @@
-{ lib
-, nixosTests
-, stdenv
-, fetchFromGitLab
-, inih
-, libdrm
-, libinput
-, libxkbcommon
-, meson
-, ninja
-, pkg-config
-, scdoc
+{
+  lib,
+  nixosTests,
+  stdenv,
+  fetchFromGitLab,
+  inih,
+  libdrm,
+  libinput,
+  libxkbcommon,
+  meson,
+  ninja,
+  pkg-config,
+  scdoc,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "unl0kr";
-  version = "2.0.0";
+  version = "3.2.0";
 
   src = fetchFromGitLab {
     domain = "gitlab.com";
-    owner = "cherrypicker";
-    repo = "unl0kr";
+    owner = "postmarketOS";
+    repo = "buffybox";
```

Comment:
Isn't this repo archived as well? It's now at https://gitlab.postmarketos.org/postmarketOS/buffybox/

Reply:
Doesn't look like it, plus that's where unl0kr migrated

Reply:
Right, it's on their own instance now, gonna correct once tests come through